File: sdk/client-ts/src/___PLAYGROUND___/relayer/get-address.ts
import { begin } from "../helpers";

export const getAddress = async () => {
    const context = await begin();

    console.log("Getting relayer address...");
    const address = await context.relayer.address();
    console.log("Relayer address:", address);

    await context.end();
};

getAddress().then(() => console.log("get-address done"));

File: sdk/python/src/__PLAYGROUND__/relayer/get-address.py
import asyncio

from __PLAYGROUND__.helper import begin, end


async def getAddress():
    rrelayer_node = None

    try:
        _, relayer, _, rrelayer_node = await begin()

        print("Getting relayer address...")
        address = await relayer.address()
        print("Relayer address:", address)
    except Exception as e:
        print("getAddress failed:", e)
    finally:
        if rrelayer_node is not None:
            end(rrelayer_node)


if __name__ == "__main__":
    asyncio.run(getAddress())
    print("get-address done")
